xml/book_cloud_suppl.xml

```diff
@@ -1406,6 +1406,13 @@
       The <systemitem class="resource">python-magnumclient</systemitem> is installed.
      </para>
     </listitem>
+            <listitem>
+     <para>
+    Install the <package>openstack-magnum-k8s-image-x86_64</package> package. 
+    This provides a Kubernetes-based virtual machine, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>, that provides OpenStack Magnum Container 
```

Comment:
I've got a few modifications for this sentence (newly added stuff highlighted):

This *package* provides a virtual machine *image with Kubernetes pre-installed*, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>*. OpenStack Magnum uses this image when creating clusters with its `k8s_opensuse_v1` driver.*